packages/angular/cli/models/command-runner.ts

```diff
@@ -357,7 +357,7 @@ export function parseOptions(args: string[], optionsAndArguments: Option[]) {
 }
 
 // Find a command.
-function findCommand(map: CommandMap, name: string): CommandLocation | null {
+export function findCommand(map: CommandMap, name: string): CommandLocation | null {
```

Comment:
We need parse the command from commandMap, too. One current usage is to check whether a command is a schematic command. 
We could 
1. have our own method, but it essentially does the same thing with this method.  
2. Use import to get the command constructor as we previously did. But since the external cli already has this dynamic loading, it would be better to have the same mechanism with external cone.

Reply:
Updated the PR according to discussion.